File: src/targets/c/libcurl/fixtures/short.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "GET");
<<<<<<< HEAD
curl_easy_setopt(hnd, CURLOPT_URL, "https://httpbin.org/anything");
=======
curl_easy_setopt(hnd, CURLOPT_URL, "http://mockbin.com/har");
>>>>>>> upstream/master

CURLcode ret = curl_easy_perform(hnd);

File: src/targets/node/axios/fixtures/short.js
<<<<<<< HEAD
const axios = require('axios');

const options = {method: 'GET', url: 'https://httpbin.org/anything'};

axios
  .request(options)
  .then(function (response) {
    console.log(response.data);
  })
  .catch(function (error) {
    console.error(error);
  });
=======
const axios = require('axios').default;

const options = {method: 'GET', url: 'http://mockbin.com/har'};

try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}
>>>>>>> upstream/master
